tasklist/qa/util/src/main/java/io/camunda/tasklist/qa/util/TestContainerUtil.java

```diff
@@ -398,11 +398,11 @@ public GenericContainer createTasklistContainer(
       String dockerImageName, String version, TestContext testContext) {
     tasklistContainer =
         new GenericContainer<>(String.format("%s:%s", dockerImageName, version))
-            .withExposedPorts(8080)
+            .withExposedPorts(8080, 9600)
```

Comment:
@romansmirnov we need to do the same changes in the charts and docs again, as we did for Operate I guess?

Reply:
Yes, Houssain is aware of the required changes in the helm charts.